src/mod.ts

```diff
@@ -1,6 +1,7 @@
 export {Replicache} from './replicache';
 export {TransactionClosedError} from './transaction-closed-error';
 
+export type {Hash} from './hash';
```

Comment:
No.

We should not expose this in the public API

Reply:
I see.  So just src/mod.ts is the public API, correct?  The other mod.ts files are for cross directory export/import?

Reply:
That is correct